javatests/arcs/core/host/LifecycleParticles.kt

```diff
@@ -229,3 +229,34 @@ class ReadWriteAccessParticle : AbstractReadWriteAccessParticle() {
         }
     }
 }
+
+class PipelineProducerParticle : AbstractPipelineProducerParticle() {
+    override fun onFirstStart() {
+        handles.sngWrite.store(Value("sng"))
+        handles.colWrite.store(Value("col"))
```

Comment:
A side note outside of this PR: it seems like it'd be good to have a `storeAll()` function for collections.

Reply:
That may be true, but note that these are to different handles being written to.

Reply:
Yes, I recognize that. This came to my mind more so in the `onFirstStart` example where I store three entities in a collection (and requires 3 calls to store).